Makefile

```diff
@@ -9,10 +9,11 @@ CURRENT_DIR := $(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 # Clones the knora-api git repository
 .PHONY: clone-knora-stack
 clone-knora-stack:
-	@git clone --branch v13.0.0-rc.16 --single-branch --depth 1 https://github.com/dasch-swiss/knora-api.git $(CURRENT_DIR)/.tmp/knora-stack
+	@rm -rf $(CURRENT_DIR)/.tmp/knora-stack
+	@git clone --branch v13.0.0-rc.21 --single-branch --depth 1 https://github.com/dasch-swiss/knora-api.git $(CURRENT_DIR)/.tmp/knora-stack
```

Comment:
If the Knora version is updated for the e2e tests, then the corresponding version of dsp-js should be used too. However, the dsp-js version for Knora rc.21 has not been released yet.

Otherwise, the unit test data from dsp-js does match the version of Knora to run the e2e tests against.

Please see section "Publish a new version to npm" in README.md (the one in the project root)

Reply:
Thanks @tobiasschweizer 
Back to rc.16. 
Could you please finish the review now.